### DISCUSSION TITLE
Use of native PyBind11?

### DISCUSSION THREAD

**Discussion by Trophime** [feelpp/feelpp#1270] [General] [answered]
Can we use native PyBind11 to build pyfeel?

**Comment by prudhomm** [comment, marked as answer]
why is this needed? @Trophime

**Reply by prudhomm** [reply]
@Trophime it is an old question, why do you need the _native_ pybind11 ?

**Reply by prudhomm** [reply]
@Trophime pybind11 as submodule helps us get in sync with upstream whatever the local packaged version of pybind11. It is more reliable that way albeit with some extra effort.
Note that pybind11 is header only no conflicts whatsoever at link time with possible system install
